src/plugins/intel_npu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -4,969 +4,9 @@
-	_skipRegistry.addPatterns("Skip tests that can not wrong when DRIVER is default compiler type", {
-                ".*OVClassLoadNetworkTestNPU.LoadNetworkHETEROWithDeviceIDNoThrow.*",
-                ".*MatMulTransposeConcatTest.*"
-        });
-
-        // [Tracking number: E#121043]
-        _skipRegistry.addPatterns("Compiler adapter is not extracting network name from metadata", {
-                ".*smoke_BehaviorTests/OVClassCompiledModelGetPropertyTest.GetMetricNoThrow_NETWORK_NAME.*"
-        });
-
-        // [Tracking number: E#121347]
-        _skipRegistry.addPatterns("Error message for empty model from stream must be changed to have \"device xml header\"", {
-                ".*smoke_BehaviorTests/OVClassCompiledModelImportExportTestP.smoke_ImportNetworkThrowWithDeviceName.*",
-                ".*smoke_Hetero_BehaviorTests/OVClassCompiledModelImportExportTestP.smoke_ImportNetworkThrowWithDeviceName.*"
-        });
-
-        // [Tracking number: E#121448]
-        _skipRegistry.addPatterns(devices.has3700() && backendName.isZero(), "CID from driver 31.0.100.1937 doesn't support certain properties", {
-                ".*smoke_BehaviorTests/OVClassCompiledModelPropertiesTests.canCompileModelWithPropertiesAndCheckGetProperty.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                ".*smoke_BehaviorTests/OVClassCompileModelWithCorrectPropertiesTest.CompileModelWithCorrectPropertiesTest.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                ".*smoke_BehaviorTests/OVClassCompiledModelGetPropertyTest_MODEL_PRIORITY.GetMetricNoThrow.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                ".*smoke_BehaviorTests/OVClassCompiledModelGetPropertyTest_MODEL_PRIORITY.GetMetricNoThrow.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                ".*smoke_BehaviorTests/OVClassCompiledModelGetPropertyTest_EXEC_DEVICES.CanGetExecutionDeviceInfo.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                ".*smoke_BehaviorTests/OVCompileModelGetExecutionDeviceTests.CanGetExecutionDeviceInfo.*NPU_DYNAMIC_SHAPE_TO_STATIC.*",
-                // COMPILATION_NUM_THREADS
-                ".*smoke_BehaviorTests/OVCompiledModelPropertiesDefaultSupportedTests.CanCompileWithDefaultValueFromPlugin.*"
-        });
-
-        _skipRegistry.addPatterns("NPU cannot set properties for compiled models", {
-                ".*OVClassCompiledModelSetCorrectConfigTest.canSetConfig.*"
-        });
-
-        // [Tracking number: CVS-139118]
-        _skipRegistry.addPatterns("Failing runtime model tests", {
-                ".*OVCompiledModelGraphUniqueNodeNamesTest.CheckUniqueNodeNames.*",
-                ".*OVExecGraphSerializationTest.ExecutionGraph.*"
-        });
-
-        return _skipRegistry;
-    }();
-    // clang-format on
-
-    std::vector<std::string> matchingPatterns;
-    const auto currentTestName = getCurrentTestName();
-    matchingPatterns.emplace_back(skipRegistry.getMatchingPattern(currentTestName));
-
-    return matchingPatterns;
+    return std::vector<std::string>{};
```

Comment:
Do we expect no disabled tests?

It is just not very clear what is the motivation for the change.

Reply:
We expect disabled tests, the motivation here is primarily to do a general cleanup. To try and declutter the skip_configs while also migrating tests from the old repo.